docs/src/components/Guidelines/index.tsx

```diff
@@ -1,161 +1,146 @@
 import React from "react";
 import styled, { css } from "styled-components";
 import { CheckCircle, CloseCircle } from "@kiwicom/orbit-components/icons";
-import { Stack, Text, Heading } from "@kiwicom/orbit-components";
-import { imageWrapperClass } from "gatsby-remark-images/constants";
+import {
+  Stack,
+  Text,
+  Heading,
+  Grid,
+  useMediaQuery,
+  mediaQueries as mq,
+} from "@kiwicom/orbit-components";
+import useTheme from "@kiwicom/orbit-components/lib/hooks/useTheme";
```

Comment:
Every time you see something that isn't being exported from the `@kiwicom/orbit-components` root please instead of importing from `lib` add that export (in a separate commit) and use it 🙏